### PR TITLE
Ignore alpha when comparing colors in MessageEmbed

### PR DESCRIPTION
**Pull request by babbaj**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: #893

## Description

see title and #893

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java [hunk 1]
```diff
@@ -398,7 +398,7 @@ public boolean equals(Object obj)
             && Objects.equals(videoInfo, other.videoInfo)
             && Objects.equals(footer, other.footer)
             && Objects.equals(image, other.image)
-            && Objects.equals(color, other.color)
+            && Objects.equals(color & 0x00FFFFFF, other.color & 0x00FFFFFF)
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: Can we change this to `==` instead of equals?

Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]:
```suggestion
            && (color & 0xFFFFFF) == (other.color & 0xFFFFFF)
```

Comment (babbaj) [reply src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: the extra zeroes are there for readability

Comment (babbaj) [reply src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: it makes it more clear that youre zeroing the first byte

Comment (MinnDevelopment) [reply src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: I disagree with that, besides that its also inconsistent with how we do this everywhere in JDA. We never do padding with 0's for this and its not needed. That isn't the important part of the suggested change, the part that actually matters is the change from `equals` to `==`.

**Pull request merged**